pandas/tests/sparse/test_frame.py

```diff
@@ -1231,7 +1232,10 @@ def test_from_to_scipy_object(spmatrix, fill_value):
     dtype = object
     columns = list('cd')
     index = list('ab')
-    tm.skip_if_no_package('scipy', max_version='0.19.0')
+    import scipy
+    if (spmatrix is scipy.sparse.dok_matrix and LooseVersion(
+            scipy.__version__) >= LooseVersion('0.19.0')):
+        pytest.skip("dok_matrix from object does not work in SciPy >= 0.19")
```

Comment:
This is a slightly different take on the existing code, which was the only place I saw the ``max_version`` argument being used. Rather than re-build that in the new module, I changed this around to be more explicit about the issue encountered when running this for SciPy v > 0.19. 

Note that all of the scenarios in this test issue the warning that``"object dtype is not supported by sparse matrices"``